src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -204,22 +204,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     packages references.
     ============================================================
     -->
-  <UsingTask TaskName="Microsoft.NET.Build.Tasks.CollectResolvedSDKReferencesDesignTime"
-             AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
-  
-  <Target Name="CollectSDKReferencesDesignTime"
-          Returns="@(_SDKReference)"
-          DependsOnTargets="ResolvePackageDependenciesDesignTime">
+  <UsingTask TaskName="Microsoft.NET.Build.Tasks.CollectSDKReferencesDesignTime"
+         AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
 
-    <CollectResolvedSDKReferencesDesignTime
-          ResolvedSdkReferences="@(SdkReference)"
-          DependenciesDesignTime="@(_DependenciesDesignTime)">
+  <Target Name="CollectSDKReferencesDesignTime"
+          Returns="@(_SDKReference)">
```

Comment:
PackageReference here coming from evaluation. and should be available. However  ResolvedPackageReference items indeed need to be accessed after collection target, however that is what i am trying to avoid here since that target reads assets file and is slow. In addition , we woul drun each of those targets 1 per target framework inthe project which would be even slower :)...